solr/core/src/java/org/apache/solr/core/SolrXmlConfig.java

```diff
@@ -168,10 +168,6 @@ public static NodeConfig fromConfig(
     configBuilder.setBackupRepositoryPlugins(
         getBackupRepositoryPluginInfos(root.get("backup").getAll("repository")));
     configBuilder.setClusterPlugins(getClusterPlugins(loader, root));
-    // <metrics><hiddenSysProps></metrics> will be removed in Solr 10, but until then, use it if a
-    // <hiddenSysProps> is not provided under <solr>.
-    // Remove this line in 10.0
-    configBuilder.setHiddenSysProps(getHiddenSysProps(root.get("metrics")));
```

Comment:
@HoustonPutman you marked this as deprecated...    Does that mean I can remove config.setHiddenSystProps too?    I *think* so..   we keep the concept of hidden though for the redaction capablity right?